content/index.zh-cn.md

```diff
@@ -0,0 +1,14 @@
+---
+date: "2016-11-08T16:00:00+02:00"
+title: "欢迎"
+weight: 10
+toc: false
+draft: false
+type: "default"
+---
+
+# Gitea - Git with a cup of tea
```

Comment:
https://github.com/go-gitea/website/pull/15
and how to enter the right language according browser setting?